core/src/replay_stage.rs

```diff
@@ -3367,15 +3367,11 @@ impl ReplayStage {
             Measure::start("generate_new_bank_forks_get_slots_since");
         let next_slots = blockstore
             .get_slots_since(&frozen_bank_slots)
-            .expect("Db error");
+            .expect("Expected Blockstore::get_slots_since() to succeed");
         generate_new_bank_forks_get_slots_since.stop();
 
         // Filter out what we've already seen
-        trace!("generate new forks {:?}", {
-            let mut next_slots = next_slots.iter().collect::<Vec<_>>();
-            next_slots.sort();
-            next_slots
-        });
+        trace!("generate new forks {:?}", next_slots.clone().sort());
```

Comment:
They slots we pass to `get_slots_since()` are the keys from a `HashMap`. As such, they will be in no particular order. So, we do still need the sort in order to ensure the slots are in increasing order in this `trace!` statement; however, wondering if we can drop the `.sort()`, which would also allow dropping of the `.clone()`